### ISSUE TITLE
Sorting with verbatim fields does not work

### ISSUE THREAD

**Issue by dstei** [plk/biber#263]
Sorting with a custom sorting template using the eprint field does not work:

```Latex
\documentclass{article}
\usepackage{filecontents}
\usepackage[backend=biber]{biblatex}

\DeclareSortingTemplate{eprint}{
    \sort[direction=descending]{\field{eprint}}
}

\begin{filecontents}{references.bib}
@article{A01,
    author = {Author, A.},
    year = {2014},
    title = {Alpha},
    eprint = {1402.1354},
}
@article{B02,
    author = {Buthor, B.},
    year = {2016},
    title = {Bravo},
    eprint = {1602.1354},
}
@article{C03,
    author = {Cuthor, C.},
    year = {2015},
    title = {Charlie},
    eprint = {1504.1354},
}
\end{filecontents}

\addbibresource{references.bib}

\begin{document}
\newrefcontext[sorting=eprint]
\nocite{*}
\printbibliography
\end{document}
```
moewew suggested this issue is not limited to eprint but to all verbatim and also uri fields.

**Comment by plk** [comment]
Should be fixed in 2.13 DEV. Please note that you will also need biblatex 3.13 DEV.

**Comment by plk** [comment]
Fixed in latest 2.13 release.
